lib/ftp/src/ftp.erl

```diff
@@ -2338,8 +2338,8 @@ file_close(Fd) ->
 
 file_read(Fd) ->
     case file:read(Fd, ?FILE_BUFSIZE) of
-        {ok, Bytes} ->
-            {ok, size(Bytes), Bytes};
+        {ok, Bytes} when is_binary(Bytes) ->
```

Comment:
The `is_binary/1` test is not strictly necessary here, because `file:read/2` is guaranteed to never return a bitstring.

Reply:
Thinking about this, `file:read` can return `string() | binary()`, so I think that the when should be re-added?

Reply:
Yes.